TAP version 13
    # Subtest: exceed_plan
    1..1
    ok 1 - (unnamed test)
not ok 1 - exceed_plan ___/# time=[0-9.]+(ms)?/~~~
  ---
  {"at":{"column":5,"file":"test/test/catch-tap-throws.js","line":5},"error":"test count exceeds plan","results":{"count":1,"ok":false,"pass":1,"plan":{"end":1,"start":1}},"source":"tap.test(function exceed_plan (t) {\n"}
  ...

not ok 2 - Error: test count exceeds plan
  ---
  {"at":{"column":7,"file":"test/test/catch-tap-throws.js","function":"exceed_plan","line":9},"count":2,"message":"Error: test count exceeds plan","plan":1,"source":"t.pass()\n","test":"exceed_plan"}
  ...
1..2
# failed 2 of 2 tests
___/# time=[0-9.]+(ms)?/~~~

